README.md

```diff
@@ -26,29 +26,29 @@ Technique of Encryption
   <h4>LSB based Image steganography:</h4>
```

Comment:
the heading here is not according to the hierarchy. Can you maybe change that?

Reply:
And the ``` ## Contributing ``` at line 99

Reply:
Done @BreadGenie